src/ucs/stats/stats.c

```diff
@@ -460,6 +474,144 @@ void ucs_stats_node_free(ucs_stats_node_t *node)
     }
 }
 
+static size_t
+ucs_stats_agrgt_sum_clsid_alloc(ucs_stats_node_t *node)
+{
+    ucs_stats_filter_node_t *filter_node = node->filter_node;
+    ucs_stats_node_t *temp_node;
+    uint32_t counter_index;
+    size_t *offset;
+    ucs_stats_aggrgt_sum_counter_name_t *element_cnt_names;
```

Comment:
"element" or "counter"

Reply:
Renamed to counter_names:

    ucs_stats_aggrgt_sum_counter_name_t *counter_names;